crates/solidity/outputs/cargo/build/src/main.rs

```diff
@@ -17,7 +17,8 @@ use solidity_language::GrammarConstructorDslV1;
 fn main() -> Result<()> {
     // Generate files in the source crate:
     {
-        let grammar = Grammar::from_dsl_v1();
+        let grammar = Grammar::from_dsl_v2();
```

Comment:
Looks like both tests codegen `/crates/solidity/outputs/cargo/tests/build.rs` and sanctuary codegen `crates/solidity/testing/sanctuary/src/main.rs` are still using DSL v0 .. We probably didn't run into issues as we kept them in sync.